_posts/2021-07-05-newsletter.md

```diff
@@ -0,0 +1,199 @@
+
+Here is a small and non-representative list of Lua plugins:
+
+- [Plenary](https://github.com/nvim-lua/plenary.nvim) -- A lua library of useful utilities for developing Neovim plugins (some of which will later be integrated into core).
+- [Packer](https://github.com/wbthomason/packer.nvim) -- A use-package like plugin manager with support for plugin dependencies, lazy-loading, and installing luarocks.
+- [Telescope](https://github.com/nvim-telescope/telescope.nvim) -- A highly extendable fuzzy finder over lists.
+- [Gitsigns](https://github.com/lewis6991/gitsigns.nvim) -- A Lua (teal) plugin for showing and interacting with changes on files a git repository (asynchronously).
+- [Nvim-compe](https://github.com/nvim-lua/completion-nvim) -- An auto completion framework for various sources, including Neovim's builtin LSP client.
+- [Nvim-dap](https://github.com/mfussenegger/nvim-dap) -- A debug adapter protocol implementation for step-through debugging of your code.
+- [Colorizer](https://github.com/norcalli/nvim-colorizer.lua) -- A high-performance color highlighter for Neovim without any external dependencies.
+- [Formatter](https://github.com/mhartington/formatter.nvim) -- A plugin for asynchronously executing external formatting tools on the current buffer or range.
+- [Hop.nvim](https://github.com/phaazon/hop.nvim) - Easy-motion without messing with your buffer.
+- [Neogit](https://github.com/TimUntersberger/neogit) - Magit but for Neovim.
+
+Not all of these are actually written in Lua: there are many other languages (some of which are typed) that compile to Lua, e.g.,
+- [Fennel(Lisp)](https://fennel-lang.org) via [Aniseed],(https://github.com/Olical/aniseed) or [Fennel-nvim](https://github.com/jaawerth/fennel-nvim),
+- [Teal](https://github.com/teal-language/tl) (a typed dialect of Lua),
+- [Amulet ML](https://amulet.works),
+- [MoonScript](https://github.com/leafo/moonscript) via [nvim-moonmaker](https://github.com/svermeulen/nvim-moonmaker) ,
+- [Vim9script](https://github.com/tjdevries/vim9jit) (you read that right).
+
+### Lua config
+
+It is also possible to write user configuration in Lua: If there is an `init.lua`, it is read _instead of_ `init.vim` (these cannot coexist, and having both in your config directory will give an error), and `.lua` files in runtime directories (`plugin/`, `colorscheme/`, `after/` etc.) are sourced in addition to (after) Vimscript files. (Yes, this means that you can write your color scheme as a `.lua` file!) Note that this is entirely optional and not needed to enjoy the new functions introduced in Neovim 0.5; furthermore, not every Vimscript configuration option has a fully native Lua equivalent yet. Extending the native API to cover these as well is part of the goal for Neovim 0.6.
```

Comment:
move the dot after the parentheses `files. (Yes,`

Reply:
Isn't that what is already there?

(I can remove the parentheses completely, it might be distracting here.)